internal/api/front/perses/src/app/shared/service/config.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, of } from 'rxjs';
+import { catchError, tap } from 'rxjs/operators';
+
+export class Configuration {
+  activateDynamicAutoCompletion = true;
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ConfigService extends Configuration {
+  private static readonly CONFIG_PATH = '/assets/config.json';
```

Comment:
So since front-end and back-end are merged in aingle binary, I think it will be more interesting to deliver the configuration through a dedicated endpoint served by the backend.

But it's a bit out of the scope of this PR. And not yet necessary from my point of view.

Reply:
Exactly that was the plan in the next step of this PR. But yeah if we say that dynamic autocompletion is forced to disabled for now, because we need to think how to link it with datasources, then I'll don't need any configuration for now.

Reply:
config service removed for now